kubera/authentication/create_user.go

```diff
@@ -0,0 +1,85 @@
+package authentication
+
+import (
+	"encoding/json"
+
+	"github.com/go-resty/resty/v2"
+	conn "github.com/mayadata-io/kubera-api-testing/kubera/connection"
```

Comment:
avoid alias `conn` , we can directly refer it by `connection` as it is used at only one place in whole file.

Reply:
done